docs/gettsim_objects/variables.rst

```diff
@@ -26,9 +26,9 @@ household.
 +-------------------------+----------------------------------------------+-------------+
 | _`wohnort_ost`          | Dummy: Living in former East Germany         | BoolSeries  |
 +-------------------------+----------------------------------------------+-------------+
-| _`prv_krankenv`         | Dummy: In private health insurance           | BoolSeries  |
+| _`in_priv_krankenv`        | Dummy: In private health insurance           | BoolSeries  |
 +-------------------------+----------------------------------------------+-------------+
-| _`prv_rente_beitr_m`    | Monthly private pension contribution         | FloatSeries |
+| _`priv_rente_beitr_m`   | Monthly private pension contribution         | FloatSeries |
```

Comment:
```suggestion
| _`priv_rentenv_beitr_m`   | Monthly private pension contribution         | FloatSeries |
```
Should be fully consistent with `ges_rentenv_...`

Reply:
> It is correct below, of course

What exactly did you mean by that?

Otherwise, the renaming is done.

Reply:
That `nv` is missing from `priv_rentenv_beitr_m`, but it is correct at other places in the code. I initially thought the latter was not the case, hence the initial comment; I added the second part ex post.